app/views/words/_payjoin-design-flow.md

```diff
@@ -17,34 +17,35 @@ The resulting flow differs from the existing Bitcoin UI Kit in three key ways. I
 
 First, when Bob scans the merchant's QR, the transaction overview shows it's a payjoin with a colorful toggle. There is some information to show a privacy-preserving transaction will be used by default.
```

Comment:
"There is some information to show a privacy-preserving transaction will be used by default." what are you referring to?

Reply:
the payjoin toggle